app/controllers/katello/concerns/api/v2/registration_commands_controller_extensions.rb

```diff
@@ -28,7 +28,7 @@ module ApiPieExtensions
         update_api(:create) do
           param :registration_command, Hash do
             param :activation_key, String, desc: N_('Activation key for subscription-manager client, required for CentOS and Red Hat Enterprise Linux. For multiple keys use `activation_keys` param instead.'), deprecated: true
-            param :activation_keys, Array, desc: N_('Activation keys for subscription-manager client, required for CentOS and Red Hat Enterprise Linux. Required only if host group has no activation keys.')
+            param :activation_keys, Array, desc: N_('Activation keys for subscription-manager client, required for CentOS and Red Hat Enterprise Linux. Required only if host group has no activation keys or if host group is not assigned.')
```

Comment:
"or if no host group is given"

Would this be better?

Reply:
I'm fine with either :+1: